docker-builds/reference-smart/clients.json

```diff
@@ -1,12 +1,25 @@
 [{
-  "client_id": "app-demo",
-  "client_name": "S4S Patient App Demo",
-  "client_uri": "https://app.demo.syncfor.science",
+  "client_id": "research-app",
+  "client_name": "S4S Reference Research App",
+  "client_uri": "https://research-app.demo.syncfor.science",
   "scope": "patient/*.read",
   "client_secret": "demo-secret-s4s",
   "redirect_uris": [
-    "http://localhost:9001/fhir-app/",
-    "https://app.demo.syncfor.science/authorized"
+    "http://localhost:8000/authorized/",
```

Comment:
Ok, let me totally drive you crazy: let's leave on port, 9001, but with a path or /authorized. (To explain my own insanity: I was imagining re-using the entire redirect URL of http://localhost:8000/fhir-app, but I see now from your example that would make local testing a huge pain.)